portable/ThirdParty/GCC/RP2040/port.c

```diff
@@ -530,7 +530,11 @@ void vClearInterruptMaskFromISR( __attribute__( ( unused ) ) uint32_t ulMask )
 
 void vYieldCore( int xCoreID )
 {
-    configASSERT(xCoreID != portGET_CORE_ID());
+    /* Remove warning about unused parameters. */
+    ( void ) xCoreID;
```

Comment:
Suggest to make use of this macro here.
configASSERT_DEFINED

Reply:
Updated. Because it doesn't impact performance, I updated the comment to describe the reason.